c2corg_ui/__init__.py

```diff
@@ -28,6 +28,8 @@ def main(global_config, **settings):
     config.add_route('waypoints_view', '/waypoints/{id}/{lang}')
     config.add_route('waypoints_index_default', '/waypoints')
     config.add_route('waypoints_index', '/waypoints/list*filters')
```

Comment:
Done at a39ded8
* removed the remainder match ``*filters``
* renamed routes ``*_index_default`` to ``*_index`` => URLs like ``waypoints/list/foo/bar`` (as in v5) are no longer supported, only ``waypoints?foo=bar``. In some way it hurts a bit the backward compatibility from v5 but I prefer we avoid having a bunch of "legacy" routes in the pyramid code.